pxr/imaging/plugin/hdRpr/mesh.cpp

```diff
@@ -614,11 +627,24 @@ void HdRprMesh::Sync(HdSceneDelegate* sceneDelegate,
                                 }
                             }
                         }
+
+                        if (!m_opacitySamples.empty()) {
+                            if (newPoint) {
+                                for (int sampleIndex = 0; sampleIndex < m_uvSamples.size(); ++sampleIndex) {
+                                    if (sampleIndex >= m_opacitySamples.size() || pointIndex >= m_opacitySamples[sampleIndex].size()) {
+                                        TF_WARN("Failed verification sampleIndex >= m_opacitySamples.size() || pointIndex >= m_opacitySamples[sampleIndex].size()");
```

Comment:
This is a very "developer-y" error message. Should it be a warning or debug?  Or better explained

Reply:
Actually, I just copy-pasted the code for displayColor... It's for debugging. I changed the message for a better explanation.